server/status_test.go

```diff
@@ -242,34 +249,44 @@ func getRequest(t *testing.T, ts TestServer, path string) []byte {
 	if err != nil {
 		t.Fatal(err)
 	}
-	req, err := http.NewRequest("GET", testContext.RequestScheme()+"://"+ts.ServingAddr()+path, nil)
-	if err != nil {
-		t.Fatal(err)
-	}
-	req.Header.Set(util.AcceptHeader, util.JSONContentType)
-	resp, err := httpClient.Do(req)
-	if err != nil {
-		t.Fatal(err)
-	}
-	defer resp.Body.Close()
-	if resp.StatusCode != http.StatusOK {
-		t.Errorf("unexpected status code: %v", resp.StatusCode)
-	}
-	returnedContentType := resp.Header.Get(util.ContentTypeHeader)
-	if returnedContentType != util.JSONContentType {
-		t.Errorf("unexpected content type: %v", returnedContentType)
-	}
-	body, err := ioutil.ReadAll(resp.Body)
-	if err != nil {
-		t.Fatal(err)
+	// TODO(bram) #1940: Remove retry logic.
+	for r := retry.Start(retryOptions); r.Next(); {
+		req, err := http.NewRequest("GET", testContext.RequestScheme()+"://"+ts.ServingAddr()+path, nil)
+		if err != nil {
+			t.Log(err)
+			continue
+		}
+		req.Header.Set(util.AcceptHeader, util.JSONContentType)
+		resp, err := httpClient.Do(req)
+		if err != nil {
+			t.Log(err)
+			continue
+		}
+		defer resp.Body.Close()
+		if resp.StatusCode != http.StatusOK {
+			t.Logf("unexpected status code: %v", resp.StatusCode)
+			continue
+		}
+		returnedContentType := resp.Header.Get(util.ContentTypeHeader)
+		if returnedContentType != util.JSONContentType {
+			t.Logf("unexpected content type: %v", returnedContentType)
+			continue
+		}
+		body, err := ioutil.ReadAll(resp.Body)
+		if err != nil {
+			t.Log(err)
+			continue
+		}
+		return body
 	}
-	return body
+	t.Fatalf("There was an error retrieving %s", path)
+	return []byte("")
 }
 
 // startServer will start a server with a short scan interval, wait for
 // the scan to complete, and return the server. The caller is
 // responsible for stopping the server.
-// TODO(Bram): Add more nodes.
+// TODO(bram): Add more nodes.
```

Comment:
no, but there were more instances of `TODO(bram)` than there were of `TODO(Bram)`